addons/website/static/tests/tours/snippet_image_gallery.js

```diff
@@ -0,0 +1,46 @@
+odoo.define("website.tour.snippet_image_gallery", function (require) {
+"use strict";
+
+const tour = require("web_tour.tour");
+const wTourUtils = require("website.tour_utils");
+
+tour.register("snippet_image_gallery", {
+    test: true,
+    url: "/",
+}, [
+    wTourUtils.clickOnEdit(),
+    wTourUtils.dragNDrop({
+        id: "s_image_gallery",
+        name: "Image Gallery",
+}), wTourUtils.clickOnSnippet({
+    id: 's_image_gallery',
+    name: 'Image Gallery',
+}), {
+    content: "Click on Remove all",
+    trigger: "we-button:has(div:contains('Remove all'))",
+}, {
+    content: "Click on Add Images",
+    trigger: "span:contains('Add Images')",
+}, {
+    content: "Click on the first new image",
+    trigger: ".o_select_media_dialog img[title='s_default_image.jpg']",
+}, {
+    content: "Click on the second new image",
+    trigger: ".o_select_media_dialog img[title='s_default_image2.jpg']",
+}, wTourUtils.addMedia(),
```

Comment:
I think it would be better to keep one step per line, it makes it easier to read. Nice that you used the util though 👍🏻